ProcessMaker/Http/Controllers/Auth/LoginController.php

```diff
@@ -55,6 +56,19 @@ public function showLoginForm()
     {
         $manager = App::make(LoginManager::class);
         $addons = $manager->list();
+        $default = Setting::where('key', 'sso.default.login')->first();
+
+        // Redirectind to default login
+        if (!empty($default) && !empty($addons->toArray())) {
+            $position = $default->getAttribute('config');
+            $element = $default->getAttribute('ui')->elements[$position];
+            $drivers = $addons->toArray()[0]->data['drivers'];
```

Comment:
Use laravel helper to get the head of array https://laravel.com/docs/10.x/helpers